src/sentry/notifications/utils/participants.py

```diff
@@ -230,12 +234,35 @@ def determine_eligible_recipients(
         if team:
             return {team}
 
+    elif target_type == ActionTargetType.RELEASE_MEMBERS:
+        return get_release_committers(project, event)
+
     else:
         return get_owners(project, event)
 
     return set()
 
 
+def get_release_committers(project: Project, event: Event) -> Sequence[User]:
+    # get_participants_for_release seems to be the method called when deployments happen
+    # supposedly, this logic should be fairly, close ...
+    # why is get_participants_for_release so much more complex???
```

Comment:
I think it's mostly because it handled checking user notification settings. These are handled by `get_recipients_by_provider` later on in this workflow, so we don't need to worry as much here. We should double check this later on, but since we're just sending to our team initially it's not as big of a deal.